doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -354,6 +354,9 @@ nRF Desktop
   * The pin control (:kconfig:option:`CONFIG_PINCTRL`) dependency to the :ref:`nrf_desktop_wheel`.
     The dependency simplifies accessing **A** and **B** QDEC pins in the wheel module's implementation.
     The pin control is selected by the QDEC driver (:kconfig:option:`CONFIG_QDEC_NRFX`).
+  * Added support for the nRF54L15 PDK with the ``nrf54l15pdk_nrf54l15_cpuapp`` board target.
+    The DK can act as a sample mouse or keyboard.
```

Comment:
PDK or DK?

Reply:
Looking at the other comments: `PDK` - updated